nextcord/guild.py

```diff
@@ -3463,3 +3467,197 @@ async def register_application_commands(self, *commands: BaseApplicationCommand)
+        exempt_roles: List[Role] = MISSING,
+        exempt_channels: List[GuildChannel] = MISSING,
+    ) -> AutoModerationRule:
+        """|coro|
+
+        Create an auto moderation rule.
+
+        You must have the :attr:`~Permissions.manage_guild` to be able to do this.
+
+        Parameters
+        ----------
+        name: :class:`str`
+            The name of this auto moderation rule.
+        event_type: :class:`EventType`
+            The event type that this rule should start finding. Default to :attr:`EventType.message_send`.
+        trigger_type: :class:`TriggerType`.
+            The trigger type of this rule.
+        keyword_filters: List[:class:`str`]
+            The words that this rule should filter.
+        presets: List[:class:`KeywordPresetType`]
+            The preset keywords that this rule should filter.
+        notify_channel: :class:`abc.GuildChannel`
+            The channel that Discord will send an alert to this channel when the rule is triggered.
+            Either this or `timeout_seconds` must be provided.
+        timeout_seconds: :class:`int`
+            The seconds to timeout the user that triggered this rule.
+            Either this or `notify_channel` must be provided.
+        block_message: :class:`bool`
+            Whether or not the message that triggered the rule should be blocked when this rule is triggered. Default is True.
+        enabled: :class:`bool`
+            Whether or not this rule is enabled. Default is True.
+        exempt_roles: List[:class:`Role`]
+            The roles that should not be affected by this rule.
+        exempt_channels: List[:class:`abc.GuildChannel`]
+            The channels that should not be affected by this rule.
+
+        Returns
+        -------
+        :class:`AutoModerationRule`
+            The rule that was created.
+
+        Raises
+        -------
+        Forbidden
+            You do not have proper permissions to do this.
+        InvalidArgument
+            You specified both ``presets`` and ``keyword_filters``.
+        """
+        params = {
+            "name": name,
+            "event_type": event_type.value,
+            "trigger_type": trigger_type.value,
+            "enabled": enabled,
+        }
+        if (keyword_filters is not MISSING) and (presets is not MISSING):
+            raise InvalidArgument(
+                "Cannot pass keyword_filters and presets to create_automod_rule()"
+            )
+        if (keyword_filters is not MISSING) or (presets is not MISSING):
+            params["trigger_metadata"] = {}
+        if keyword_filters is not MISSING:
+            params["trigger_metadata"]["keyword_filters"] = keyword_filters
+        if presets is not MISSING:
+            params["trigger_metadata"]["presets"] = [preset.value for preset in presets]
+        if (
+            (notify_channel is not MISSING)
+            or (timeout_seconds is not MISSING)
+            or (block_message is True)
+        ):
```

Comment:
These extra brackets are all unrequired?

Reply:
for  one it thing at 3634 and 3638 it was because i needed to debug

Reply:
*for the thing

Reply:
for 3644 -> 3648 slap that emotional damage to pre-commit instead >:((( *cries*

Reply:
Your 3 comments don't explain why you need them? Why do you need these brackets? They serve no purpose from my view



> *for the thing

Because we can read your mind about what this thing is..

Reply:
i mean, for the brackets

Reply:
What do those numbers even mean

Reply:
see automod thread on discord